mmv1/third_party/terraform/services/compute/resource_compute_disk_test.go.tmpl

```diff
@@ -1644,6 +1668,61 @@ resource "google_compute_disk" "foobar" {
 `, diskName, storagePoolUrl)
 }
 
+func TestExpandStoragePoolUrl_withEmptyConfigProjectAndZone(t *testing.T) {
```

Comment:
Please also add tests for the following cases:

- project & zone are pulled from Config instead of data
- project & zone are not available on data or Config.